code/game/objects/items/devices/taperecorder.dm

```diff
@@ -79,20 +75,7 @@
 			return
 	..()
 
-
-/obj/item/taperecorder/verb/ejectverb()
-	set name = "Eject Tape"
-	set category = "Object"
-
-	if(usr.incapacitated())
-		return
-	if(!mytape)
-		return
-
-	eject(usr)
-
-
-/obj/item/taperecorder/update_icon_state()
+/obj/item/taperecorder/update_icon()
```

Comment:
```suggestion
/obj/item/taperecorder/update_icon()
	. = ..()
```

Reply:
update_icon() is now moved to update_icon_state(), shouldn't need to call parent proc now

Reply:
```
/atom/proc/update_icon(updates=ALL)
	SIGNAL_HANDLER
	SHOULD_CALL_PARENT(TRUE)
```
wrong